indico/modules/rb_new/schemas.py

```diff
@@ -77,11 +78,13 @@ class Meta:
         fields = ('id', 'booking_reason', 'booked_for_name', 'room_id', 'is_accepted')
 
 
-class ReservationEventDataSchema(Schema):
+class ReservationLinkedObjectDataSchema(Schema):
     id = Number()
     title = String()
     url = String()
-    can_access = Function(lambda event: event.can_access(session.user))
+    event_title = Function(lambda obj: obj.event.title)
```

Comment:
we shouldn't be returning this information at all if the user cannot access the object

Reply:
actually, we abort early in the RH if the user has no access so we can always return True for `can_access` here since we did the proper access check in the RH